cmeutils/structure.py

```diff
@@ -130,18 +129,47 @@ def gsd_rdf(
             system = (box, A_pos)
             aq = freud.locality.AABBQuery.from_system(system)
             nlist = aq.query(
-                B_pos, {"r_max": r_max, "exclude_ii": exclude_ii}
+                B_pos, {"r_max":r_max, "exclude_ii":exclude_ii}
             ).toNeighborList()
 
             if exclude_bonded:
                 pre_filter = len(nlist)
                 nlist.filter(
-                    molecules_A[nlist.point_indices]
+                    molecules_A[nlist.point.indices]
                     != molecules_B[nlist.query_point_indices]
                 )
                 post_filter = len(nlist)
 
             rdf.compute(aq, neighbors=nlist, reset=False)
+    normalization = post_filter / pre_filter if exclude_bonded else 1
+    return rdf, normalization
+
+def get_centers(gsdfile, new_gsdfile):
+    """Calculates the centers of a trajectory.
```

Comment:
```suggestion
    """Create a gsd file containing the molecule centers from an existing gsd file.
    
```

This is maybe a picky change, but it could be worth mentioning in the one-line description that the main purpose of the function is to write a new gsd file.